src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -147,6 +170,11 @@ main() {
     echo "=== NF cache    : $DX_PROJECT_CONTEXT_ID:/.nextflow/cache/$NXF_UUID"
     echo "============================================================="
 
+    # restore cache
+    if [[ $resume == true && -n $resume_session]]; then
+      resume_session
```

Comment:
It will be little more readable if the names of the variable for the function and the argument are different.